BlogTemplate/Services/ExcerptGenerator.cs

```diff
@@ -6,8 +6,9 @@
 namespace BlogTemplate._1.Services
 {
     public class ExcerptGenerator
-    {
-        public string CreateExcerpt(string body, int maxLength)
+    {
+        public int maxLength = 140;
```

Comment:
This should be a property, not a field.  Or better yet, it should be private and set by a constructor.  This means for the service registration, you'd have to provide an instance of ExcerptGenerator so that you call the constructor appropriately.

Reply:
Also, nit: put newlines before method declarations.